### ISSUE TITLE
Improve documentation for "Using Regular Expression Search for Auto-Linking"

### ISSUE THREAD

**Issue by claell** [JabRef/user-documentation#392]
On https://docs.jabref.org/finding-sorting-and-cleaning-entries/filelinks#using-regular-expression-search-for-auto-linking, the explanation for the regular expressions is misleading. The explanation for `*` and `**` is only valid for the beginning of the line.

Probably, it is also good to reference some general regex site explaining the usage in more detail.

Took me some time to figure out my mistakes and how to properly use this (I have not used regex and wasn't sure whether normal regex is used here or some modified version).

**Comment by tobiasdiez** [comment]
I agree there is potential for improvement. Since you went through the config process just now, could you expand on the explanation by editing the docs at https://github.com/JabRef/user-documentation/edit/main/en/finding-sorting-and-cleaning-entries/filelinks.md ?

**Comment by ThiloteE** [comment]
Solution for info about RegEx: add link to https://docs.jabref.org/finding-sorting-and-cleaning-entries/search#advanced
